go.mod

```diff
@@ -84,3 +83,5 @@ require (
 	gopkg.in/yaml.v3 v3.0.0-20210106172901-c476de37821d
 	honnef.co/go/tools v0.1.0 // indirect
 )
+
+replace go.temporal.io/api => github.com/wxing1292/api-go v1.0.1-0.20210305184909-513103793126
```

Comment:
Don't forget to remove this.

Reply:
the plan is to get an consensus, then i will update the proto repos